src/main/java/uk/gov/hmcts/divorce/citizen/event/CitizenSubmitApplication.java

```diff
@@ -77,7 +77,7 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
         }
 
         State state;
-        if (caseDataCopy.getHelpWithFeesAppliedForFees().toBoolean()) {
+        if (caseDataCopy.getHelpWithFeesAppliedForFees() != null && caseDataCopy.getHelpWithFeesAppliedForFees().toBoolean()) {
```

Comment:
can front end set this to No by default ?

Reply:
`helpWithFeesNeedHelp` always gets set to `Yes`/`No` while `helpWithFeesAppliedForFees` only gets set once the first question is `Yes`. Perhaps we should be either checking if `helpWithFeesNeedHelp` 
 is true or if `helpWithFeesReferenceNumber` is not null and not empty

Reply:
```suggestion
        if (caseDataCopy.getHelpWithFeesNeedHelp().toBoolean() && caseDataCopy.getHelpWithFeesAppliedForFees().toBoolean()) {
```

Reply:
I've switched this to use `helpWithFeesNeedHelp` instead which will always be set.